app/exawind/CMakeLists.txt

```diff
@@ -6,3 +6,9 @@ target_link_libraries(${EXAWIND_EXE_NAME} PRIVATE
   yaml-cpp)
 
 target_include_directories(${EXAWIND_EXE_NAME} SYSTEM PRIVATE ${YAML_CPP_INCLUDE_DIR})
+
+if(CXX_FILESYSTEM_HAVE_FS)
+  target_compile_definitions(${EXAWIND_EXE_NAME} PRIVATE AMR_WIND_HAS_STD_FILESYSTEM)
```

Comment:
Instead of `AMR_WIND_HAS_STD_FILESYSTEM` maybe it should be `EXAWIND_HAS_STD_FILESYSTEM` ?

Reply:
Haha yeah I wasn't paying attention.